scripts/methods.rb

```diff
@@ -60,15 +75,28 @@ def map_data(row, headings)
     when 'IFSC CODE'
       index = HEADINGS_INSERT.find_index 'IFSC'
       data[index] = row[heading_index]
-      
+
     else
       data[index] = row[heading_index] if index
     end
   end
   data
 end
 
+def ensure_data_folder_exists
```

Comment:
This is already covered by the bootstrap script: https://github.com/razorpay/ifsc/blob/master/scripts/bootstrap.sh#L10